R/LearnerRegrKM.R

```diff
@@ -32,12 +32,30 @@ LearnerRegrKM = R6Class("LearnerRegrKM", inherit = LearnerRegr,
     initialize = function() {
       ps = ParamSet$new(list(
         ParamFct$new("covtype", default = "matern5_2", levels = c("gauss", "matern5_2", "matern3_2", "exp", "powexp"), tags = "train"),
+        ParamUty$new("coef.trend", default = NULL, tags = "train"),
+        ParamUty$new("coef.cov", default = NULL, tags = "train"),
+        ParamUty$new("coef.var", default = NULL, tags = "train"),
         ParamDbl$new("nugget", tags = "train"),
         ParamLgl$new("nugget.estim", default = FALSE, tags = "train"),
-        ParamFct$new("type", default = "SK", levels = c("SK", "UK"), tags = "predict"),
         ParamDbl$new("nugget.stability", default = 0, lower = 0, tags = "train"),
+        ParamUty$new("noise.var", default = NULL, tags = "train"),
+        ParamFct$new("estim.method", default = "MLE", levels = c("MLE", "LOO"), tags = "train"),
+        ParamUty$new("penalty", default = NULL, tags = "train"),
+        ParamFct$new("optim.method", default = "BFGS", levels = c("BFGS", "gen"), tags = "train"), # gen requires rgenoud package
```

Comment:
Can we check for this param in the internal train function and add a `requireNamespace()` call?